### PR TITLE
Remove `baseUrl` and use `invocationTarget` in `request()`.

### PR DESCRIPTION
**Pull request by dlongley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -422,13 +431,13 @@ given URL to perform a read operation.
...
-**Kind**: instance method of [<code>ZcapClient</code>](#ZcapClient)  
-**Returns**: <code>Promise.&lt;object&gt;</code> - - A promise that resolves to an HTTP response.  
+**Kind**: instance method of [<code>ZcapClient</code>](#ZcapClient)
+**Returns**: <code>Promise.&lt;object&gt;</code> - - A promise that resolves to an HTTP response.
 
 | Param | Type | Description |
 | --- | --- | --- |
 | options | <code>object</code> | The options to use. |
-| options.url | <code>string</code> | The relative URL to invoke the   Authorization Capability against. |
+| options.url | <code>string</code> | The URL to invoke the   Authorization Capability against. |
```
Comment (msporny) [inline README.md:440]:
```suggestion
| options.url | <code>string</code> | The URL to invoke the Authorization Capability against. |
```

File: README.md [hunk 2]
```diff
@@ -144,11 +141,15 @@ retrieved from somewhere using application-specific code and then passed
 to the `read` method.
 
 ```js
-const url = '/my-account/items/123';
-const capability = await getCapabilityFromDatabase({url}); // defined by your code
+const url = 'https://zcap.example/my-account/items/123';
+// defined by your code
+const capability = await getCapabilityFromDatabase({url, /* other */});
```
Comment (msporny) [inline README.md:146]: It's not clear what "other" is supposed to be... happens multiple times in this PR (and examples).

**Review by msporny — APPROVED** [review]
Two minor nits, other than that LGTM.

File: README.md (hunk 2 above)
Comment (dlongley) [reply README.md:146]: Well, it's "defined by your code".

File: README.md (hunk 1 above)
Comment (dlongley) [reply README.md:440]: Those things were already there -- I'm not going to mess with them now. I don't know if they are from autogeneration code or if they are important or what.

**Pull request merged**